Dockerfile

```diff
@@ -1,8 +1,57 @@
-FROM geonode/django
-MAINTAINER GeoNode development team
+FROM python:2.7.9
```

Comment:
Probably https://github.com/GeoNode/geonode-project/blob/docker/Dockerfile#L1 it missed `geonode/django` rather than `terranodo/django`.

However we maintain such images in the official [docker hub](https://hub.docker.com/u/geonode/) where `geonode/django` is built from our django-docker github repo and definitively inherits from your proposed base `python:2.7.9`